doc/Advanced-configuration.md

```diff
@@ -1,24 +1,32 @@
-For advanced configuration use the following files: `ejabberd.cfg`, `vm.args` and `app.config`. The first one contains all pure XMPP settings, the second affects Erlang VM behaviour (performance tuning, node name) and third usually has to be modified to change low-level logging parameters. Since you've gotten this far, we assume you're already familiar with Erlang syntax.
+For advanced configuration use the following files:
+
+* `ejabberd.cfg` for pure XMPP settings, 
```

Comment:
`for pure MongooseIM settings`. I know this doesn't sound well. On the other hand saying this is only about XMPP settings is not fully true.